### ISSUE TITLE
[A11y] Desktop/Mobile Search Results List: Image Links

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1835]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results List: https://staging.oregondigital.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q%5B%5D=test

Failed Instance: 
link-name: Ensures links have discernible text

Image 1
Path: #document_9z902z939 > .search-result-wrapper.row > .col-sm-4.col-md-3.col-xs-12 > .list-thumbnail > a
Snippet:
`<a data-context-href="/catalog/9z902z939/track?locale=en&amp;per_page=8&amp;search_id=1414" href="/concern/images/9z902z939?locale=en"><img src="/downloads/h989r329k?file=thumbnail"></a>`

Image 2
Path:#document_kw52j8067 > .search-result-wrapper.row > .col-sm-4.col-md-3.col-xs-12 > .list-thumbnail > a
Snippet:
`<a data-context-href="/catalog/kw52j8067/track?locale=en&amp;per_page=8&amp;search_id=1414" href="/concern/generics/kw52j8067?locale=en"><img src="/assets/work-ff055336041c3f7d310ad69109eda4a887b16ec501f35afc0a547c4adb97ee72.png"></a>`

Image 3
Path: #document_1544bp11q > .search-result-wrapper.row > .col-sm-4.col-md-3.col-xs-12 > .list-thumbnail > a
Snippet:
`<a data-context-href="/catalog/1544bp11q/track?locale=en&amp;per_page=8&amp;search_id=1414" href="/concern/generics/1544bp11q?locale=en"><img src="/downloads/70795771g?file=thumbnail"></a>`

### Suggested Fix

Fix the following:

- Element is in tab order and does not have accessible text

Fix ONE of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

**Comment by CGillen** [comment]
Fixed by #2037
